classes/order/Order.php

```diff
@@ -1259,6 +1259,7 @@ public function setInvoice($use_existing_payment = false)
 				UPDATE `'._DB_PREFIX_.'order_detail`
 				SET `id_order_invoice` = '.(int)$order_invoice->id.'
 				WHERE `id_order` = '.(int)$order_invoice->id_order);
+            Cache::clean('objectmodel_'.'OrderDetail'.'_*');
```

Comment:
You do not have to concatenate elements when you have only string.

Please replace it with 
```php
Cache::clean('objectmodel_OrderDetail_*');
```